### ISSUE TITLE
Sound not restoring to previous level after call end

### ISSUE THREAD

**Issue by quack-o** [andyholmes/valent#767]
### Current Behavior

When i am playing a media on my laptop with valent running on it and i recieve a call on android running kdeconnect , the video pauses as expected. but after call is over and media resumes, the sound goes to 15% .

### Expected Behavior

Sound levels should be as before the call 

### Desktop

Other (please specify)

### Other Desktop

hyprland

### Operating System

Arch

### Installed from

Distribution Package

### Version

1.0.0.alpha

### Devices

Valent (Desktop Linux)

**Comment by andyholmes** [comment]
I've noticed this too, but I'm not sure the cause. I recorded the sequence of packets for each event sequence, but it seems the "hang-up" event gets lost or very delayed sometimes.

Some in-depth debugging is probably necessary to find what the actual cause is here.

**Comment by quack-o** [comment]
if "hang-up" event is getting lost, then how media is getting resumed ? that has to be the trigger, no ?

I can try to debug, if you can tell me where to start with and how to capture events

**Comment by andyholmes** [comment]
Sorry, your issue title said the sound was not restoring. So it's set to plower the volume and then pause, and it's restarting the music but it's not raising the volume?

**Comment by Iss-in** [comment]
yes. playback resumes after call is cut, but volumes remains at previously lowered level

**Comment by isshin1** [comment]
@andyholmes https://github.com/andyholmes/valent/pull/1027
can you please take a look
